### ISSUE TITLE
How to use Swal.queue?

### ISSUE THREAD

**Issue by webcoderkz** [avil13/vue-sweetalert2#104]
There's an example in the docs:

```js
Swal.queue([{
  title: 'Your public IP',
  confirmButtonText: 'Show my public IP',
  text:
    'Your public IP will be received ' +
    'via AJAX request',
  showLoaderOnConfirm: true,
  preConfirm: () => {
    return fetch(ipAPI)
      .then(response => response.json())
      .then(data => Swal.insertQueueStep(data.ip))
      .catch(() => {
        Swal.insertQueueStep({
          icon: 'error',
          title: 'Unable to get your public IP'
        })
      })
  }
}])
```

But seems like this package only supports `Swal.fire()`? Is there a way to use a queue?

**Comment by requiem202** [comment]
I imported Swal from 'sweetalert2' directly and it worked fine.

**Comment by avil13** [comment]
Fixed in (d3720322261)
